examples/development_lifecycle/development_lifecycle/decks.py

```diff
@@ -236,6 +236,29 @@ def table_renderer() -> None:
     )
 
 
+# %% [markdown]
+# :::{figure} https://raw.githubusercontent.com/flyteorg/static-resources/main/flytesnacks/user_guide/flyte_decks_source_code_renderer.png
+# :alt: Source code renderer
+# :class: with-shadow
+# :::
+# #### Source Code Renderer
+# Converts a Source Code into HTML
+# %%
+from flytekitplugins.deck.renderer import SourceCodeRenderer
+import inspect
+
+
+@task(disable_deck=False)
```

Comment:
```suggestion
@task(enable_deck=True)
```

Reply:
I use `disable_deck=False` because align 
https://github.com/flyteorg/flytesnacks/blob/26a61f672d990e48c42962e4265a3ed1e878e780/examples/development_lifecycle/development_lifecycle/decks.py#L231-L236

curious, what is the difference between using `enable_deck=True` and `disable_deck=False`?

Reply:
got it